src/data_hub/button_callback.js

```diff
@@ -0,0 +1,35 @@
+// Callback when a user presses the button to update labels in the scatter plot Bokeh figure
+
+const selectedData = lasso_selection_source.data;
+const label = label_select.value;
+const labelColumn = label_column_select.value;
+
+console.log("Will write to these rows:", selectedData.ids);
+console.log("Will write this label:", label);
+console.log("Will write to this column:", labelColumn);
+
+const rowIds = selectedData.ids;
+
+if (typeof window.deeporigin !== "undefined") {
+    // Reset selections for all the rows in question
+    const resetChanges = rowIds.map(rowId => ({
+        rowId: rowId,
+        fieldChangeEvents: [{
+            columnId: labelColumn,
+            newValue: { selectedOptions: [] } 
+        }]
+    }));
+
+    deeporigin.dataHub.primaryDatabase.editRows({ changes: resetChanges });
```

Comment:
Hmm I see how you're intending to use this. I think we need to return a promise here and you need to wait for it to complete, else you may get the conflict message.

Reply:
aaaah. i see. ok, let's fix this in the next PR. i want to see if it works at all